core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/JavaDocParser.java

```diff
@@ -220,11 +221,12 @@ private void appendHtml(StringBuilder sb, Node node) {
                     sb.append(BACKTICK);
                     break;
                 case BOLD_NODE:
-                case EMPHASIS_NODE:
+                case STRONG_NODE:
                     sb.append(STAR);
                     appendHtml(sb, childNode);
                     sb.append(STAR);
                     break;
```

Comment:
I assume this makes it so that `<strong>` is rendered correctly by our Javadoc doclet?

Reply:
Yes. I thought it was the problem initially as it wasn't handled at all. And then I had the good idea to have a look at the actual javadoc :).

I fixed the tests.